workshop-resources/keyboard-navigation/event-reviews/2022-11-21-jupyteraccessibility.md

```diff
@@ -0,0 +1,222 @@
+# Collaborative keyboard navigation review
+
+Please add your findings as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request) so you are credited in the commit history!
+
+## Keyboard navigation accessibility review of [Jupyter accessibility documentation](https://jupyter-accessibility.readthedocs.io/en/latest/index.html)
+
+Reviewed via based on [this link we'll all be working off of](https://jupyter-accessibility.readthedocs.io/en/latest/index.html).
+
+- Review date:
+- Operating system:
+- Browser: 
+- Keyboard language(s) used:
+- Any other details (ie. dependencies, github repo, etc.):
+
+### Authors and contents
+
+- Content order: (add author(s))
+- Areas to navigate: (add author(s))
+- Keyboard/tab traps: (add author(s))
+- Skip links: (add author(s))
+- Interactive Areas: (add author(s))
+- Focus: (add author(s))
+- Mixed input: (add author(s))
+- Keyboard shortcuts: (add author(s))
+- Additional authors: (add author(s))
+
+## Review
+
+For this review, participants select a test they would like to complete and apply it to as many areas or states of the interface as they want in the given time.
+
+### General notes
+
+This is the place for anything a reviewer thinks is important that doesn't belong below. This could be something out of scope of the review, something that will impact all other tests, or the like.
+
+Notes:
+
+### Content order
+
+When accessed via keyboard, the content order is logical. ([WCAG 2.2 Focus order](https://www.w3.org/TR/WCAG22/#focus-order))
+
+1. From the top of the page, press the Tab key repeatedly until reviewer reaches the end of content.
+2. Reviewers will know it is the end because if they press Tab once more their keyboard focus will return to the browser.
+3. If the reviewer cannot make it to the end of content, please take note of where the focus gets stuck.
+
+Does the content order follow reviewer expectations based on reading the content?
+Result:
```

Comment:
Kind of - the order is as follows:

1. Left sidebar: logo -> search bar -> sidebar items (top to down)
2. Top left menu: icons left to right
3. Table of contents: TOC items top to bottom
4. Main content: links as they appear top to bottom -> Next navigation button	

- I would expect Areas to go left to right in a left-to-right language 
- At no given point I accessed the footer